service/routes.py

```diff
@@ -248,26 +248,67 @@ def list_items(shopcart_id):
 
 
 ######################################################################
-# READ A ITEM FROM A SHOPCART
+# READ AN ITEM FROM A SHOPCART
 ######################################################################
 @app.route("/shopcarts/<int:shopcart_id>/items/<int:item_id>", methods=["GET"])
 def get_items(shopcart_id, item_id):  # pylint: disable=unused-argument
     """
     Retrieve a single Item
 
-    This endpoint will return a Item based on it's id
+    This endpoint will return an Item based on it's id
     """
-    app.logger.info("Request to Retrieve a item with id [%s]", item_id)
+    app.logger.info("Request to Retrieve an item with id [%s]", item_id)
 
-    # Attempt to find the Item and abort if not found
-    item = Item.find(item_id)
```

Comment:
i dont think was wrong, see the sample https://github.com/nyu-devops/sample-accounts/blob/a313a5cc5346c79377624e319b1ae1547f03794e/service/routes.py#L243
I think we should keep it this way because its a lot simpler (item ID is all thats needed to find an item since thats unique, the shopcart ID isn't needed first)
I've tested it out with thunderclient and the item retrieval is fine